src/components/Main/Main.js

```diff
@@ -30,6 +32,10 @@ export class MainComponent {
 
   addEventListeners() {
     this.sidebarList.addEventListener("click", this.handleListClick.bind(this));
+    this.artistsLink.addEventListener(
+      "click",
+      this.handleArtistsView.bind(this)
```

Comment:
bind this function in the constructor

Reply:
for now it is ok to bind it here. Later, when we will be using frameworks - we will also use ESnext class properties proposal to make this better

Reply:
fixed